dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops

```diff
@@ -639,5 +639,6 @@
 
 7359=No any ASM service available.
 
-7360=Database password was encrypted by deprecated algorithm. Reencrypt it by `passwordUpdate.sh` from eclipselink.zip bundle
+7360=Database password was encrypted by deprecated algorithm.
```

Comment:
Please remove this change.... this change will undo the changes requested by ID during the last up date from 4.0.2 to 4.0.3.... see comments in : https://github.com/OpenLiberty/open-liberty/pull/28824